geode-wan/src/distributedTest/java/org/apache/geode/internal/cache/wan/CacheClientNotifierDUnitTest.java

```diff
@@ -265,11 +265,9 @@ public static void createClientWithLocator(int port0, String host, String region
       CacheServerTestUtil.enableShufflingOfEndpoints();
     }
 
-    AttributesFactory factory = new AttributesFactory();
+    RegionFactory factory = cache.createRegionFactory(RegionShortcut.LOCAL);
```

Comment:
I'm not sure if this matters, but the regions created before and after your changes here are not the same.

The new code creates this region:
```
[vm4] RMI TCP Connection(1)-127.0.0.1: CacheClientNotifierDUnitTest.createClientWithLocator region=org.apache.geode.internal.cache.LocalRegion[path='/testNormalClient2MultipleCacheServer_PR';scope=LOCAL';dataPolicy=NORMAL; concurrencyChecksEnabled]
```
The old code creates this region:
```
[vm3] RMI TCP Connection(1)-127.0.0.1: CacheClientNotifierDUnitTest.createClientWithLocator region=org.apache.geode.internal.cache.DistributedRegion[path='/testNormalClient2MultipleCacheServer_PR';scope=DISTRIBUTED_NO_ACK';dataPolicy=NORMAL; concurrencyChecksEnabled]
```

Reply:
Good catch @boglesby!. I don't think it matters for the test but the goal of this PR is to get rid of `AttributesFactory`, not to modify the actual tests, so I've changed the region creation to reflect what it was doing before. 
I'll merge the changes once CI is green, thanks again for reviewing!.